_checks/md_check_hook.rb

```diff
@@ -1,6 +1,6 @@
 # This Jekyll hook runs Markdown linter (https://github.com/markdownlint/markdownlint)
 # on .md files that git tracks as 'modified' ('git ls-files -m').
-# '_checks/md_style' sets a style (https://github.com/markdownlint/markdownlint/blob/master/docs/creating_styles.md)
+# '_checks/styles/md_style' sets a style (https://github.com/markdownlint/markdownlint/blob/master/docs/creating_styles.md)
```

Comment:
should this be `_checks/styles/style-rules-dev`?

Reply:
fixed